orangecontrib/snom/widgets/owpreprocessimage.py

```diff
@@ -273,6 +317,31 @@ def init_attr_values(self, data):
         self.attr_x = self.xy_model[0] if self.xy_model else None
         self.attr_y = self.xy_model[1] if len(self.xy_model) >= 2 else self.attr_x
 
+    def init_mask_values(self, data):
+        domain = data.domain if data is not None else None
+        self.mask_value_model.set_domain(domain)
+        if self.data is not None:
+            self.update_mask_value_items()
+
+    def update_mask_value_items(self):
+        self.cb_mask_value.clear()
+        try:
+            self.cb_mask_value.addItems(
+                list(self.data.domain[self.mask_attr_value].values)
+            )
+            self.cb_mask_value.setCurrentIndex(0)
+            self.mask_group_value = 0
+            # Need to update manually
+            self.set_mask_from_selection()
+        except:
```

Comment:
On this repository lint is not optional, it is strict, because otherwise all future tests will fail too. Why are you catching an exception here, and in such a large set of line.

Exceptions should be limited to the smallest problematic section and specific. If you really have no control of the type of the exception, a comment in the code describing the reason is needed, and also, an instruction for the linter to ignore the issue.